plugin.video.aswim/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.aswim"
+       name="Adult Swim"
+       version="3.0.1"
+       provider-name="t1m">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.t1mlib" version="1.0.18"/>
+    <import addon="inputstream.adaptive" version="2.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource"
+            library="default.py">
+        <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Adult Swim</summary>
+    <description lang="en_GB">Adult Swim</description>
+    <disclaimer lang="en">Feel free to use this script. For information visit the wiki.</disclaimer>
+    <platform>all</platform>
+    <language>en</language>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
Missing Icon and Fanart parameters, Also can you add a fanart and perhaps bump the icon to 512x512. THX

Reply:
Low life thief